src/main/java/seedu/address/logic/commands/AddSpecialisationCommand.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_DOCTORS;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.doctor.Doctor;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Adds a prescription to an existing appointment in the database.
```

Comment:
`prescription to an existing appointment` change to `specialisation for a doctor`

Reply:
Ok noted, Thanks pointing out